kolibri/content/api.py

```diff
@@ -231,3 +231,13 @@ class FileViewset(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return models.File.objects.all()
+
+class ChannelFileSummarySerializer(viewsets.ModelViewSet):
+    serializer_class = serializers.ChannelFileSummarySerializer
+
+    def get_queryset(self):
+        aggs = models.File.objects.aggregate(
+            total_files = Count('pk'),
+            total_file_size = Sum('file_size')
+        )
+        return [aggs]
```

Comment:
@jonboiser probably best to do this on the serializer - the queryset object should be a queryset (or, as you discovered, at the very least an iterable, hence wrapping it in a list works).